sources/packages/api/src/route-controllers/institution-locations/institution-locations.controller.ts

```diff
@@ -161,6 +169,32 @@ export class InstitutionLocationsController extends BaseController {
     });
   }
 
+  /**
+   * Get all active application of a location in an institution
+   * with application_status is completed.
+   * @param locationId location id.
+   * @returns Student active application list of an institution location.
+   */
+  @HasLocationAccess("locationId")
```

Comment:
The resource that we are fetching here is application. An application which is filtered by location and status.
Shouldn't this be in application.controller.ts ?

Reply:
Yes we are fetching the application, but it is fetched with respect to institution-location, so we are having it in the institution-location controller.